### PR TITLE
Added basic (mostly generated) styling

### PR DESCRIPTION
**Pull request by ramonvermeulen**
### Checklist

- [ ] When adding a Github handle to be tracked that is not yours, request the user to approve the PR

### CODE REVIEW

**Comment by JCZuurmond** [comment]
Thanks, @ramonvermeulen. Is the css required to be defined in the template? Or can we link a style sheet via an url, just like we do at the top of the document?

**Comment by ramonvermeulen** [comment]
> Thanks, @ramonvermeulen. Is the css required to be defined in the template? Or can we link a style sheet via an url, just like we do at the top of the document?

Yes, I only don't have access to change the workflow, since we then need to copy the `assets` folder to `/site/build`.

**Comment by JCZuurmond** [comment]
The workflows are in `.github/workflows/..`, you should have access to that. 

You can move the assets inside the build folder. I build the website locally using:

```
python _site/generate_html.py github_contributions.duckdb _site/build/index.html
```

That only works with assets inside the build folder.

**Comment by JCZuurmond** [comment]
And could you add to a README in `_site/README.md` how you generate the javascript and CSS? In case we want to change it.

**Comment by ramonvermeulen** [comment]
> The workflows are in `.github/workflows/..`, you should have access to that.
> 
> You can move the assets inside the build folder. I build the website locally using:
> 
> ```
> python _site/generate_html.py github_contributions.duckdb _site/build/index.html
> ```
> 
> That only works with assets inside the build folder.

Yes, but if I try to push changes to one of the workflows files I get an error on that: 
```
[remote rejected] ramon/styling-improvements -> ramon/styling-improvements (refusing to allow a Personal Access Token to create or update workflow `.github/workflows/deploy_to_github_pages.yml` without `workflow` scope)
```

File: _site/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Site
+
+The site is a static html page, generated with jinja using the python `generate_html.py` script. 
+The css is for the most part generated using the datatables [__theme creator__](https://datatables.net/manual/styling/theme-creator).
```
Comment (JCZuurmond) [inline _site/README.md:4]: Thanks!

**Comment by timotk** [comment]
@ramonvermeulen Really like the styling :) 

Two things:
- You can try using the `font-family` found on Xebia's website
- Could you make the text of the table content black? It helps readability. Headers of course can stay purple!

For example (probably download the font and use it as your asset):
```css
@font-face {
  font-family: Proxima;
  src: url('https://xebia.com/wp-content/themes/xebia-theme/css/fonts/proximanova-regular-webfont.woff2')
          format('woff2'),
      url('https://xebia.com/wp-content/themes/xebia-theme/css/fonts/proximanova-regular-webfont.woff')
          format('woff');
}

@font-face {
  font-family: Proxima;
  font-weight: 500;
  src: url('https://xebia.com/wp-content/themes/xebia-theme/css/fonts/proximanova-semibold-webfont.woff2')
          format('woff2'),
      url('https://xebia.com/wp-content/themes/xebia-theme/css/fonts/proximanova-semibold-webfont.woff')
          format('woff');
}

@font-face {
  font-family: Proxima;
  font-weight: 700;
  src: url('https://xebia.com/wp-content/themes/xebia-theme/css/fonts/proximanova-bold-webfont.woff2')
          format('woff2'),
      url('https://xebia.com/wp-content/themes/xebia-theme/css/fonts/proximanova-bold-webfont.woff')
          format('woff');
}

html {
  font-family: 'Proxima', sans-serif;
}
```

**Comment by ramonvermeulen** [comment]
> 

@timotk  Nice suggestions, just changed the font and text color. Still not the greatest design, but at least some styling in place :')

**Review by JCZuurmond — APPROVED** [review]
Thanks, @ramonvermeulen! I am going to merge it. @jochemloedeman is planning to work on a dashboard this Friday

**Pull request merged**